rust/src/mirror.rs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use crate::downloads::read_content_from_link;
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+
+pub const MIRROR_URL: &str =
+    "https://raw.githubusercontent.com/SeleniumHQ/selenium/trunk/common/mirror/selenium";
```

Comment:
In the current PR, no, that value is hardcoded and cannot be changed by the user. 

But I agree that would be desirable. My [proposal for rich configuration](https://github.com/SeleniumHQ/selenium/issues/11690) is three levels: CLI parameters, configuration file, and environment variable. 

For this specific URL, it could be easy to include a CLI flag to configure that URL (e.g., `--mirror-url <url>`) in Rust. But that would imply changing the bindings to pass that URL from options.

IMO, implementing a configuration file/envs mechanism in Rust could ease global configuration capabilities.

Reply:
Well, we also have the URLs for ChromeDriver and others in the same way. We should probably leave it as it is and revisit later.